system-tests/tia-maven/maven-dump-local-project/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>tia-maven</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <tia.teamscale.fake.port>${env.TEAMSCALE_PORT}</tia.teamscale.fake.port>
+        <tia.agent.version>${env.AGENT_VERSION}</tia.agent.version>
+        <surefire.includeJUnit5Engines>junit-jupiter</surefire.includeJUnit5Engines>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.8.2</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.teamscale</groupId>
+            <artifactId>impacted-test-engine</artifactId>
+            <version>${tia.agent.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.teamscale</groupId>
+                <artifactId>teamscale-maven-plugin</artifactId>
+                <version>${tia.agent.version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-tia-unit-test</goal>
+                            <goal>prepare-tia-integration-test</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <teamscaleUrl>http://localhost:${tia.teamscale.fake.port}</teamscaleUrl>
```

Comment:
Besides the scenario for the concrete customer that you are building this for, most others would probably not want to specify Teamscale credentials when not upload coverage and not retrieving impacted tests. Would it be easy to adjust the validation so that it does not require them in this case?

Reply:
Adjusted.